climate_categories/data/IPCC2006_PRIMAP.yaml

```diff
@@ -3204,6 +3204,7 @@ categories:
     - 3A
     info:
       gases:
+      - N2O
```

Comment:
This appeared up when I ran `data_generation/IPCC2006_PRIMAP.py`.

Reply:
Yeah, I just missed re-generating after fixing this.